clients/extension/scripts/popup.js

```diff
@@ -80,32 +82,36 @@ function find(query, context = document) {
 
 /**
  * @param {Error} err
- * @return {HTMLAnchorElement}
+ * @return {HTMLButtonElement}
  */
 function buildReportErrorLink(err) {
   const issueBody = `
 **Lighthouse Version**: ${getLighthouseVersion()}
 **Lighthouse Commit**: ${getLighthouseCommitHash()}
 **Chrome Version**: ${getChromeVersion()}
 **Initial URL**: ${siteURL}
-**Error Message**: ${err.message}
+**Error Message**: ${err.friendlyMessage || err.message}
 **Stack Trace**:
 \`\`\`
 ${err.stack}
 \`\`\`
     `;
 
-  const url = new URL('https://github.com/GoogleChrome/lighthouse/issues/new');
-
-  const errorTitle = err.message.substring(0, MAX_ISSUE_ERROR_LENGTH);
-  url.searchParams.append('title', `Extension Error: ${errorTitle}`);
-  url.searchParams.append('body', issueBody.trim());
-
-  const reportErrorEl = document.createElement('a');
+  const reportErrorEl = document.createElement('button');
   reportErrorEl.className = 'button button--report-error';
-  reportErrorEl.href = url.href;
-  reportErrorEl.textContent = 'Report Error';
-  reportErrorEl.target = '_blank';
+  reportErrorEl.textContent = 'Copy to clipboard';
+
+  reportErrorEl.addEventListener('click', async () => {
+    try {
+      // @ts-ignore
+      await navigator.clipboard.writeText(issueBody);
+      reportErrorEl.textContent = 'Copied to clipboard';
+      setTimeout(() => {
+        reportErrorEl.textContent = 'Copy to clipboard';
+      }, 1000);
+    } catch (err) {
+    }
```

Comment:
do we need a `try/catch` since this is only in the extension UI, so will always be Chrome only?

If we do need it, we should have a `catch` fallback action, e.g. set the button text to `'Cannot copy to clipboard'` and disable the button?

Reply:
I removed the try catch, we'll add it back if we get error reports of this. As I'm unsure when chrome can't write to clipboard